Tools/MediaServer/Utility.Tests/ImageHelperTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+
+namespace Utility.Tests
+{
+    public class ImageHelperTests
+    {
+        [Test]
+        [TestCase(@"E:\Images\default.png")]
+        [TestCase(@"E:\Images\default.jpg")]
+        [TestCase(@"E:\Images\default.jpeg")]
+        public void GetImageContentTypeTest1(string path)
+        {
+            string imageContentType = ImageHelper.GetImageContentType(path);
+            Dictionary<string, string> imageMimeTypes = ImageHelper.GetImageMimeTypes();

```

Comment:
Won't work
GetImageMimeTypes is used to get test value and to get the result value, so it will also be the same.
You'd better send expected result as second param